js/src/lib/interfaces/Libraries.ts

```diff
@@ -395,6 +396,36 @@ const transformers = (model: ModelData) => {
 	}
 };
 
+const peftTask = (peftTaskType: string) => {
+	switch (peftTaskType) {
+		case "CausalLM":
+			return "CausalLM";
+		case "SEQ_2_SEQ_LM":
+			return "Seq2SeqLM";
+		case "TOKEN_CLS":
+			return "TokenClassification";
+		case "SEQ_CLS":
+			return "SequenceClassification";
+		default:
+			return undefined;	
+	}
+};
+
+const peft = (model: ModelData) => {
+	const peftBaseModel = model.config?.base_model_name;
+	const peftTaskType = model.config?.task_type;
```

Comment:
At the moment `config` is not populated, you will need to do a PR in moon landing to add the `base_model_name` and `task_type` in `moon-landing` first. You can then check with something like

```python
from huggingface_hub import model_info
model_info("Mursel/falcon-7b-instruct-finetuned")
```

Reply:
I think I already did in 6640 in moon-landing, sorry I haven't asked for review there @osanseviero